src/views/home/index.tsx

```diff
@@ -0,0 +1,184 @@
+import BigNumber from "bignumber.js";
+import { useReservesList } from "hooks/useReservesList";
+import { useAtom } from "jotai";
+import { FC, useEffect, useState } from "react";
+
+import { useWallet } from "@solana/wallet-adapter-react";
+import {
+  connectionAtom,
+  selectedPoolAtom,
+  selectedReserveAtom,
+} from "stores/globalStates";
+import {
+  SbwrModal,
+  PoolPositionModal,
+  MobileTable,
+  DesktopTable,
+} from "views/home/components";
+import { formatPoolValue, formatPoolName } from "utils/formatUtils";
+import { Error, Loader } from "components";
+import { PublicKey } from "@solana/web3.js";
+import { parseLendingMarket } from "@solendprotocol/solend-sdk/dist/state/lendingMarket";
+import { SOLEND_ADDRESSES } from "common/config";
+
+export const HomeView: FC = ({}) => {
+  const [selectedPool, setSelectedPool] = useAtom(selectedPoolAtom);
+  const [selectedReserve, setSelectedReserve] = useAtom(selectedReserveAtom);
+  const { reservesList, isLoading, isError } = useReservesList();
+  const { connected } = useWallet();
+  const [connection] = useAtom(connectionAtom);
+  const [showAPY, setShowAPY] = useState(true);
+  const [poolCreator, setPoolCreator] = useState<string | null>(null);
+
+  useEffect(() => {
+    const showPoolCreator = async () => {
```

Comment:
I'd rather have async calls in a hook and not in a useEffect in a component. I think it's nicer to keep the React component logic thin and focused on UI. The data and business logic should like in the store/hook